src/settlement-chain/NodeRegistry.sol

```diff
@@ -9,29 +9,21 @@ import { INodeRegistry } from "./interfaces/INodeRegistry.sol";
 
 import { Migratable } from "../abstract/Migratable.sol";
 
-// TODO: `nodeOperatorCommissionPercent` (and thus `MAX_BPS`) likely does not belong in this contract.
+// TODO: Make the rest of the parameters come from the parameter registry.
```

Comment:
Note that every change to the NodeRegistry means a completely new deployment in xmtpd, starting with a blank db. 
Ideally we'd finish everything that has to happen in NodeRegistry before the next deployment, so we can minimize the amount of non compatible new versions.